tests/test_interfaces.py

```diff
@@ -18,7 +21,8 @@ def test_interfaces(duthost):
         # verify no ipv4 address for each port channel member
         for member in v['members']:
             ans_ifname = "ansible_%s" % member
-            assert not host_facts[ans_ifname].has_key("ipv4")
+            pytest_assert("ipv4" not in host_facts[ans_ifname],
+                          "LAG member {} has IP address {}".format(ans_ifname, host_facts[ans_ifname]))
```

Comment:
for ip addr - host_facts[ans_ifname]["ipv4"]

Reply:
I am leaning towards show a bit more information than just ipv4.

Reply:
Changed according to your suggestion. The benefit is that the error message is more accurate.

Reply:
Sorry, I backed out the change. The message was calculated regardless of the assertion condition, so there will be key error in success cases.

Reply:
oh yes. my bad